protocol/contracts/amo/Ramos.sol

```diff
@@ -541,24 +541,18 @@ contract Ramos is IRamos, TempleElevatedAccess, Pausable {
         amoStaking.withdrawAndUnwrap(bptIn, false, address(this));
         balancerVault.exitPool(balancerPoolId, address(this), address(this), request);
 
+        unchecked {
```

Comment:
The change addressing the actual issue https://github.com/yAudit/temple-lending-report/issues/55